client/packages/coldchain/src/Monitoring/api/TemperatureBreach/api.ts

```diff
@@ -1,12 +1,12 @@
-import { FilterBy, SortBy } from '@common/hooks';
+import { FilterByWithBoolean, SortBy } from '@common/hooks';
 import { Sdk, TemperatureBreachFragment } from './operations.generated';
 import { TemperatureBreachSortFieldInput } from '@common/types';
 
 export type ListParams = {
   first: number;
   offset: number;
   sortBy: SortBy<TemperatureBreachFragment>;
-  filterBy: FilterBy | null;
+  filterBy: FilterByWithBoolean | null;
```

Comment:
the `acknowledged` filter is a boolean and to use that, the `FilterBy` needs to include the bool